### PR TITLE
Added docs about Snowplow contexts.

### PR DESCRIPTION
**Pull request by thoragudf**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…acking-plan/properties**](https://docs-k8zluldy8.vercel.app/workspace/tracking-plan/properties) |
|:-:|
|<a href="https://docs-k8zluldy8.vercel.app/workspace/tracking-plan/properties">Screenshot of /workspace/tracking-plan/properties</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/BjEqsRMUyIdHsWspNnB7qzQDZbs_.png">(view full size)</a> |







Commit 8ed2f5345c1af927dabc6618c80393ce6b8ee1ce (https://docs-k8zluldy8.vercel.app).

File: pages/workspace/tracking-plan/properties.mdx [hunk 1]
```diff
@@ -112,3 +112,78 @@ For configuring per-event and per-source you can change the overall setting for
...
+    width="60%"
+    alt="Example of a Snowplow context definition in Avo"
+  />
+</center>
+
+Here's how an individual Snowplow context is translated from what is defined in the UI to the context data structure:
+
+```json
+{
+  "data": {
```
Comment (alex-tpom6oh) [inline pages/workspace/tracking-plan/properties.mdx:132]: One more thing that every Snowplow context and the root event sent to Snowplow need to have is the pinned "schema" property, i.e. this is how it's done in BoF case https://www.avo.app/schemas/gsaQteWxmXjpUc8AGauc/events/2bUG-rWa5C

File: pages/workspace/tracking-plan/properties.mdx [hunk 2]
```diff
@@ -112,3 +112,78 @@ For configuring per-event and per-source you can change the overall setting for
...
+  }
+]
+```
+
+The array of context is then sent with the event where event call to Snowplow that looks like this:
+
+```javascript
+Snowplow.logEvent({
+  "data": {
+    "avo_event_name": "CART_UPDATED"
```
Comment (alex-tpom6oh) [inline pages/workspace/tracking-plan/properties.mdx:168]: All root properties defined outside of the snowplow context would go here, kinda like this is a root snowplow context.

**Comment by thoragudf** [comment]
Okay I made some changes @alex-tpom6oh – I'm not sure if it's better or not 😬 . Can you jump in and edit so it makes sense to a developer that know snowplow very well 😇 ?

**Pull request merged**